api/users/models.py

```diff
@@ -288,15 +288,17 @@ def send_alert_to_admin_users(subject, message):
 
     @classmethod
     def send_organisation_over_limit_alert(cls, organisation):
```

Comment:
Is this related to this PR? Or it just slipped in?

Reply:
Ah, yes, that just slipped in I think but I think it needs doing - I can't remember if I did it in another branch somewhere as well. Leave it with me...

Reply:
I've removed this from here and created a separate PR here: #888.